### PR TITLE
Update kURL upgrade docs for new K8s upgrade process

### PR DESCRIPTION
**Pull request by paigecalvert**
https://app.shortcut.com/replicated/story/75353/edit-info-about-upgrading-k8s-by-more-than-2-minor-versions-for-kurl-clusters

**Enterprise docs updates**

Updated the Kubernetes Update section to explain how you can upgrade any number of minor versions at once: https://deploy-preview-1107--replicated-docs.netlify.app/enterprise/updating-embedded-cluster#kubernetes

Adds info about how you need to download a package for air gap Kubernetes upgrades of more than one minor version: https://deploy-preview-1107--replicated-docs.netlify.app/enterprise/updating-embedded-cluster#kubernetes-multi

Add a note to the air gap upgrade procedure to describe how the script looks for the assets and prints a command to download them if they can't be found: https://deploy-preview-1107--replicated-docs.netlify.app/enterprise/updating-embedded-cluster#air-gap-environments

**Vendor docs update**:

https://deploy-preview-1107--replicated-docs.netlify.app/vendor/installer-history#about-using-inactive-installers

### CODE REVIEW

File: docs/vendor/installer-history.md [hunk 1]
```diff
@@ -8,9 +8,7 @@ Each release channel in the Replicated vendor portal saves the history of Kubern
 
 It can be useful to access the installation commands for inactive installers to reproduce an issue that a user is experiencing for troubleshooting purposes. For example, if the user's cluster is running the inactive installer version 1.0.0, then you can install with version 1.0.0 in a test environment to troubleshoot.
 
-You can also send the installation commands for inactive installers to your users as needed. One common use case for users to run inactive installers is when they are updating their cluster.
-
-For example, users might be required to run the installation command for an inactive installer to upgrade Kubernetes by more than two minor versions. This is because the Kubernetes installer supports upgrading no more than two minor versions of Kubernetes at a time. For more information about the cluster update process for enterprise users, see [Updating Kubernetes Installer Clusters](/enterprise/updating-embedded-cluster) in _Enterprise_. 
+You can also send the installation commands for inactive installers to your users as needed. For example, a user might have unique requirements for specific versions of Kubernetes or add-ons.
```
Comment (paigecalvert) [inline docs/vendor/installer-history.md:11]: >For example, a user might have unique requirements for specific versions of Kubernetes or add-ons.

Not sure if this is true. The main use case that I understood for sending inactive installer was to facilitate multi-version k8s upgrades. Now that that's not necessary, is there another common use case to call out here?

File: docs/partials/updating/_upgradePrompt.mdx [hunk 2]
```diff
@@ -1,7 +1,3 @@
...
+(Kubernetes Upgrades Only) If a Kubernetes upgrade is required, the script automatically prints a `Drain local node and apply upgrade?` prompt. Confirm the prompt to drain the local primary node and apply the Kubernetes upgrade to the control plane.
 
-   The script continues to drain and upgrade nodes sequentially. For each node, the script prints a command that you must run on the node to upgrade Kubernetes. 
-
-   After the Kubernetes upgrade on each node is complete, the script automatically updates any add-ons in your cluster.
-
-   For more information about this update process, see [About Updating Kubernetes Installer Clusters](#overview) above.
\ No newline at end of file
+   The script continues to drain and upgrade nodes sequentially. For each node, the script prints a command that you must run on the node to upgrade Kubernetes. For more information, see [About Updating Kubernetes Installer Clusters](#overview) above.
```
Comment (paigecalvert) [inline docs/partials/updating/_upgradePrompt.mdx:3]: Note: The only changes I made to this partial were for the purpose of shortening it. It seemed overly wordy.

File: docs/partials/updating/_installerRequirements.mdx [hunk 3]
```diff
@@ -1,12 +1,12 @@
-* **installer-spec-file**: If you used the `installer-spec-file` flag to pass a `patch.yaml` file when you ran the installation script previously, you must include the `installer-spec-file` flag with the same `patch.yaml` file when you rerun the installation script. This prevents the installer from overwriting any configuration from your `patch.yaml` file and making changes to the add-ons in your cluster. For example: `curl https://kurl.sh/latest | sudo bash -s installer-spec-file="./patch.yaml"`
+* **installer-spec-file**: If you used the `installer-spec-file` flag to pass a `patch.yaml` file when you installed, you must pass the same `patch.yaml` file when you upgrade. This prevents the installer from overwriting any configuration from your `patch.yaml` file and making changes to the add-ons in your cluster. For example:
```
Comment (paigecalvert) [inline docs/partials/updating/_installerRequirements.mdx:1]: Note: The only changes I made to this partial were for the purpose of shortening it. It seemed overly wordy.

File: docs/enterprise/updating-embedded-cluster.md [hunk 4]
```diff
@@ -11,26 +11,41 @@ The application vendor uses a Kubernetes installer specification file to specify
...
+⚙  Downloading assets required for Kubernetes 1.23.17 to 1.26.3 upgrade
+The following packages are not available locally, and are required:
+   kubernetes-1.24.12.tar.gz
+   kubernetes-1.25.8.tar.gz
 
-If the script detects that the version of Kubernetes in your cluster is more than two minor versions earlier than the target version, it prints an error message similar to the following: `The currently installed kubernetes version is 1.23.16. The requested version to upgrade to is 1.26.0. Kurl can only be upgraded two minor versions at time. Please install 1.25.x. first.`
+You can download them with the following command:
 
-To update Kubernetes when your currently installed version is more than two minor versions behind the target version, contact your application vendor for an additional Kubernetes installer installation script that specifies the prerequisite Kubernetes version indicated in the error message.
+   curl -LO https://kurl.sh/bundle/version/v2023.04.24-0/19d41b7/packages/kubernetes-1.24.12,kubernetes-1.25.8.tar.gz
```
Comment (paigecalvert) [inline docs/enterprise/updating-embedded-cluster.md:44]: If users wanted to download the package ahead of time and move it to their `/var/lib/kurl/assets/` directory, where could they go to download it? 

Is the idea that they should manually construct the URL?

If so, is this how they would construct it? :

```
curl -LO https://kurl.sh/bundle/version/v2023.04.24-0/19d41b7/packages/kubernetes-1.24.12,kubernetes-1.25.8.tar.gz
```
* Replace `v2023.04.24-0` with the latest version of kurl 
* Replace `kubernetes-1.24.12` with the currently running k8s version
* Replace `kubernetes-1.25.8` with the target version

Comment (paigecalvert) [reply docs/enterprise/updating-embedded-cluster.md:44]: @kyleprice ^

File: docs/vendor/installer-history.md (hunk 1 above)
Comment (paigecalvert) [reply docs/vendor/installer-history.md:11]: @kyleprice

File: docs/enterprise/updating-embedded-cluster.md (hunk 4 above)
Comment (emosbaugh) [reply docs/enterprise/updating-embedded-cluster.md:44]: I wouldnt advise that the user manually constructs this

Comment (paigecalvert) [reply docs/enterprise/updating-embedded-cluster.md:44]: Sounds good 👍 Will leave as is.

**Review by jonquil2002 — APPROVED** [review]
lgtm

**Pull request merged**